app/main.py

```diff
@@ -2,0 +1,37 @@
+class Animal:
+    animal = []
+    
+    def __init__(self, name, hidden=False, health=100):
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
```

Comment:
The attribute `alive` is not defined in the `Animal` class. It should be `animal` based on the class attribute defined. Consider changing `Animal.alive.append(self)` to `Animal.animal.append(self)`.